prime-router/src/main/kotlin/cli/helpers/HL7DiffHelper.kt

```diff
@@ -336,4 +346,27 @@ class HL7DiffHelper {
                 " Differences: $input$outputText"
         }
     }
+
+    /**
+     * helper
+     * Check that v1 and v2 are effectively both blank value
+     */
+    fun effectivelyBlank(v1: Array<ca.uhn.hl7v2.model.Type>, v2: Array<ca.uhn.hl7v2.model.Type>): Boolean {
+        return (
+            v1.size == 1 &&
+            v1[0].isEmpty &&
+            v2.isEmpty()
+        )
+    }
+
+    /**
+     * helper - heuristically check input and output are compatible types
```

Comment:
What do you mean by heuristically?

Reply:
> What do you mean by heuristically?

I mean it mimic when the diff is done by a human, or may be just use semantically blank checking...?

Reply:
Ok, I might suggest that we just take out that word since I don't think it adds much, but it's not a big deal either way.